src/index.html

```diff
@@ -21,7 +21,446 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
```

Comment:
The logo link and image are repeated in the commented-out section below. Ensure that only one version is used to avoid redundancy.